### PR TITLE
ldid-procursus: Fix memory issues when signing

### PR DESCRIPTION
**Pull request by TheRealKeto**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Various issues have been reported regarding `ldid` (from the Procursus Team) hanging when using one of its core functions (signing Mach-Os). 

[These have been reported upstream](https://github.com/ProcursusTeam/ldid/issues/41) and [the issue has been fixed](https://github.com/ProcursusTeam/ldid/pull/34). However, no new release has been made.

### CODE REVIEW

File: Formula/l/ldid-procursus.rb [hunk 1]
```diff
@@ -28,6 +28,15 @@ class LdidProcursus < Formula
 
   conflicts_with "ldid", because: "ldid installs a conflicting ldid binary"
 
+  stable do
+    # Fix memory issues with various entitlements, remove in next release
```
Comment (carlocab) [inline Formula/l/ldid-procursus.rb:32]:
```suggestion
  stable do
    url "https://github.com/ProcursusTeam/ldid.git",
        tag:      "v2.1.5-procursus7",
        revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"

    # Fix memory issues with various entitlements, remove in next release
```
You'll need to remove the `url` call outside the `stable` block too. (`version` may need the same treatment.)

Mind the spacing for the arguments to `url` (the existing spacing doesn't actually follow the style followed by other formulae).

Comment (TheRealKeto) [reply Formula/l/ldid-procursus.rb:32]: I noticed this after the CI finished, actively working on it from my end. Will push a change soon

File: Formula/l/ldid-procursus.rb [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
   license "AGPL-3.0-or-later"
-  revision 1
+  revision 2
   head "https://github.com/ProcursusTeam/ldid.git", branch: "master"
 
+  stable do
+    version "2.1.5-procursus7"
+    url "https://github.com/ProcursusTeam/ldid.git",
+      tag:      "v2.1.5-procursus7",
+      revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"
```
Comment (carlocab) [inline Formula/l/ldid-procursus.rb:12]:
```suggestion
    url "https://github.com/ProcursusTeam/ldid.git",
        tag:      "v2.1.5-procursus7",
        revision: "aaf8f23d7975ecdb8e77e3a8f22253e0a2352cef"
```
Let's follow the style used by other formulae; thanks!

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request merged**